sites/dotnet-guide/contents/tutorials/rider-essentials/debugging/index.md

```diff
@@ -0,0 +1,51 @@
+---
+type: TutorialStep
+date: 2020-06-05
+title: Debugging code with Rider
+technologies: [.net, csharp]
+topics: [debugging]
+products: [rider]
+author: khalidabuhakmeh
+subtitle: Examine the runtime behavior of your application, identify problematic code, run through code step by step.
+thumbnail: ./thumbnail.png
+longVideo: 
+    poster: ./poster.png
+    url: https://youtu.be/TODO
+---
+
+### Step through Code, Inspect what is Going On
+
+With the debugger in Rider, you can attach to a new or existing process, and place breakpoints to pause the application
```

Comment:
Do it matter whether we use `you` or `we`?

Reply:
Seems company decided on "you" so trying to move towards that after years of "we"